foundations/tying_it_all_together/conclusion.md

```diff
@@ -1,43 +1,44 @@
+Learning how to program is **much** more than just learning a language. It's about learning how to break down your problems into achievable steps, research and solve difficult problems, communicating effectively with other developers, and much more. Luckily, all these skills are transferrable to other languages and domains of software and because most languages are similar in concept and structure, moving into a different language will be much easier than learning your first.
 
 Arguments about learning the languages used in your area may sound convincing, but because a _good_ developer can transfer skills easily, what you learn isn't going to massively change outcomes of getting a job. Though, if you cannot shake that fear, do whatever you need, even if that includes researching your area to see what jobs are available locally (don't forget about remote jobs though). A good course is far more important than the language you learn first. 
 
 To help you make a decision, here are some perks of each technology:
 
 #### Ruby and Ruby On Rails: 
-* Opinionated 
-  - You don't have to make decisions on the structure of your projects. This helps when moving into projects built by others.
-  - You will learn things "The Ruby Way", which can help you write clean code in other languages as well.
-* Elegant
-  - Brings a lot of tools to help you develop things in a way that makes your code easy to read.
-  - Often times, solutions in Ruby are much more concise than other languages.
+* Convention over configuration
+  - You don't have to make decisions on the structure of your projects.
+  - Others' Ruby code tend to be similarly structured and thus easy to navigate.
+* Expressive and flexible syntax
+  - Often code can be written in a human-readable style.
+  - Easy to modify existing classes for your usecase.
 * Rapid development
-  - Rails helps you build applications very quickly, this means you can make a greater amount of interesting projects in the same time frame as others.
-* Single paradigm
-  - OOP is the Ruby way, many other languages are primarily OOP based.
+  - Rails is "batteries-included". It comes with solutions to common problems already bundled and ready to use.
+  - Rails can generate project boilerplate with it's scaffolding generators and CLI commands before you even open your editor.
+* Everything is an Object
+  - Community embraces OOP and further leads to consistency across projects.
 
 #### JavaScript/Node & Express
 * Un-opinionated
   - There is no standard way of doing things, you are going to have decide structure of projects yourself, for better or worse.
+* Large and Still Growing Community
+  - New, innovative tooling and libraries being developed all the time.
+  - Lots of new employment opportunities using JS.
 * Very fast
-  - Because JavaScript is JIT compiled, it can do calculations very quickly.
-* Verbose
```

Comment:
I think this is important to leave. People should know that JS is much more verbose in it's code.

Reply:
I disagree that es 20xx JS is *that* much more verbose that it requires users to take it into consideration when choosing their path forward. And I hard disagree that writing JS gives you any significant insight into "how computers read your code" that you wouldn't get from any other similarly high level language.

Reply:
I disagree though, there's a lot more setup with node when compared to Rails.